hub/tests/test_views.py

```diff
@@ -189,6 +246,17 @@ def test_explore_json_shader_custom_colours(self):
         self.assertEqual(response.status_code, 200)
         self.assertContains(response, "#ff0000")
 
+    def test_explore_persondata_area_type(self):
+        url = reverse("explore_json")
+        response = self.client.get(url + "?mp_election_majority__gt=1000")
+        self.assertContains(response, "South Borsetshire")
+        self.assertNotContains(response, "New South Borsetshire")
+        response = self.client.get(
+            url + "?mp_election_majority__gt=1000&area_type=WMC23"
+        )
+        self.assertContains(response, "New South Borsetshire")
+        self.assertNotContains(response, '"South Borsetshire')
```

Comment:
I assume the quoting here is intentional?

Reply:
Yup, because it's testing JSON it's checking for a JSON string that starts with `South Borsetshire` while not catching `New South Borsetshire`